integration_tests/pages/blockVisitDates/blockVisitDates.ts

```diff
@@ -9,8 +9,6 @@ export default class BlockVisitDatesPage extends Page {
     this.datePicker = new DatePickerComponent()
   }
 
-  getMessage = (): PageElement => cy.get('.moj-alert__content')
```

Comment:
Removing in favour of method in parent `Page` class (as lots of pages have MoJ alert messages)